site/content/blogs/release-v130.en.md

```diff
@@ -39,6 +39,45 @@ Copilot v1.30 brings big enhancements to help you develop more flexibly and effi
 ## Ctrl-C
 
 ## Deployment actions
+Maybe you want to run a database migration before a workload deployment, and the workload's health check depends on the
+update. Or maybe you'd like your pipeline to execute some end-to-end or integration tests after a workload deployment. These actions are
+now possible with [Copilot pipelines](../docs/concepts/pipelines.en.md)!  
```

Comment:
This is just my personal thought maybe it's bad 🤣. I feel like "copilot" is implied by that this is a copilot blog post
```suggestion
now possible with [pipelines](../docs/concepts/pipelines.en.md)!  
```

Reply:
I added that bc before it was possible with other pipelines but not copilot pipelines... 🤷🏼